src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/DefaultModelBindingContext.cs

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
```

Comment:
I think this is fine where it is. We don't have a good reason to hide it, and users might want to unit test their code.